lib/puppet/transaction/report.rb

```diff
@@ -183,7 +194,7 @@ def initialize(kind, configuration_version=nil, environment=nil, transaction_uui
     @host = Puppet[:node_name_value]
     @time = Time.now
     @kind = kind
-    @report_format = 5
+    @report_format = 6
```

Comment:
Where is the policy recorded for this thing incidentally? We have our own document outlining our versioning policy on these items, but I haven't seen such a thing for Puppet related changes.